_episodes/02-working-with-environments.md

```diff
@@ -101,13 +101,13 @@ $ conda create --name python36-env python=3.6
 ~~~
 {: .language-bash}
 
-> ## Always specify a version number for each package you wish to install
+> ## You can specify a version number for each package you wish to install
 >
 > In order to make your results more reproducible and to make it easier for research colleagues to 
-> recreate your Conda environments on their machines it is a "best practice" to always explicitly 
+> recreate your Conda environments on their machines it is sometimes a good practice to always explicitly 
```

Comment:
As discussed I agree explicit versioning can cause headaches if another package has pegged the version it depends on.